pinakes/main/catalog/models.py

```diff
@@ -122,7 +122,7 @@ class PortfolioItem(KeycloakMixin, ImageableModel, UserOwnedModel):
         help_text="The source reference this portfolio item was created from",
     )
     name = models.CharField(
-        max_length=64, help_text="Name of the portfolio item"
+        max_length=512, help_text="Name of the portfolio item"
```

Comment:
@bzwei can we use a const MAX_PORTFOLIO_ITEM_LENGTH=512 and use it in multiple places